remove-hashbang-loader.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (source) {
```

Comment:
Some of the dependencies (e.g. [JSONStream](https://github.com/dominictarr/JSONStream/blob/master/index.js)) are hybrid executable-library and have a `#! /usr/bin/env node` at the beginning of the file.